### DISCUSSION TITLE
.NET implementation - JS, AWS, Pulumi

### DISCUSSION THREAD

**Discussion by mr-j90** [realworld-apps/realworld#1010] [WIP Implementations]
Hey Friends!

Working on a .NET implementation with Bootstrap for the Frontend and .NET on the backend; also going to be using AWS for hosting and running my CI/CD pipelines from Github actions and with Puluimi running to AWS.

I will be updating this as well for anyone that would like to follow. Per the docs, I will be creating issues, and documenting the flow of the application as well. 

Happy coding! 🏗

**Comment by geromegrignon** [comment]
Hello Jordan, thanks for your contribution! 🚀
